appveyor.yml

```diff
@@ -13,7 +13,7 @@ environment:
 configuration:
   - Debug
   - Release
-version: 3.4.0.{build}
+version: 3.5.0.{build}
```

Comment:
Depending on which will be merged first, you may need to update it once more - @Sand3r- is setting the same version in his review #11.

Reply:
You can bump it up to 6.